app/models/notification.rb

```diff
@@ -19,6 +19,15 @@ class Notification < ActiveRecord::Base
 
   VALID_STATUSES = STATUS_STATES.keys
   validates :status, inclusion: { in: VALID_STATUSES }
+  validate :notification_has_at_least_one_recipient
+
+   def notification_has_at_least_one_recipient
+     # As we add more ways to choose recipients,
+     # we'll need to expand this validator
+     if departments.blank?
+       errors[:base] << "All recipients can't be blank"
```

Comment:
To make it more specific, I think we can do `errors.add(:departments, "<error message>")` instead so that it's clear what's missing.

Reply:
Perhaps premature optimization, but once I allow users to choose people individually or groups of people, the error won't be on department any more. It will be the fact that you haven't chosen a value from ANY of the fields....
The message itself could be improved though, perhaps 
`Please select someone to receive this notification from departments` and then add `or groups` to that as appropriate...